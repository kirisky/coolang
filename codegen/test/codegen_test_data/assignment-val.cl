class Main inherits IO {
  main() : Object { {
    new Main;
    new Main;
  } };
};
