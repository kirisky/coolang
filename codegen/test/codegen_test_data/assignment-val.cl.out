See the file README for a full copyright notice.
Loaded: /usr/class/cs143/cool/lib/trap.handler
x: 3
y: 4
4